.github/workflows/linter.yaml

```diff
@@ -0,0 +1,43 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
```

Comment:
If I didn't modify it, it wouldn't have passed the linter check. But sure, it could have been added after merging this branch and that probably would have been better.